client.py

```diff
@@ -5,7 +5,7 @@
 import crac_status
 from gui_constants import GuiLabel, GuiKey
 from logger import LoggerClient
-from status import Status, TelescopeStatus, PanelStatus, TrackingStatus
+from status import Status, TelescopeStatus, SwitchStatus, TrackingStatus
```

Comment:
I think maybe it's better to use ButtonStatus instead of SwitchStatus in accordance with ButtonControl